plugins/modules/katello_content_view_filter.py

```diff
@@ -126,6 +106,7 @@
     description:
       - Create an include filter
     default: False
+extends_documentation_fragment: foreman
```

Comment:
This is not an apypie module. Should this still be replaced? There's a few more of those below.

Reply:
Yes. It's a `ForemanAnsibleModule` (so nailgun, but still with the common wrapper API), which inherits from `ForemanBaseAnsibleModule` which has the common params:

https://github.com/theforeman/foreman-ansible-modules/blob/a2aac6240a419cb87876f9c91a062ecb754c6878/plugins/module_utils/foreman_helper.py#L28-L36